manifests/init.pp

```diff
@@ -104,13 +111,15 @@
       file { '/etc/cron.daily/freshclam': ensure => 'absent' }
 
       if $enable {
-        rsync { 'clamav':
-          source  => $rsync_source,
-          target  => '/var/lib/clamav',
-          server  => $rsync_server,
-          timeout => $rsync_timeout,
-          delete  => true,
-          require => Package[$package_name]
+        unless $rsync_source.empty() {
```

Comment:
Would it make more sense to have a new parameter, or even a new item in simp_options for this? It would likely be a separate ticket.

Reply:
@jeefberkey it would make the most sense to replace the `enable_freshclam` parameter with a new Enum parameter that specifies the update mechanism (including 'none').

However, it is important that that existing SIMP 6.0.X users are able to use this fix without requiring any major updates, therefore the API impact in this fix is has been kept as minimal as possible.

Reply:
@op-ct I agree with @jeefberkey . How about a Stroolean?!

`Variant[Boolean, Enum['none', 'rsync', 'freshclam']]`

Keep the existing logic for the Boolean and handle the other options as appropriate.

Reply:
@trevor-vaughan It looks like you agree more with an Enum replacement for `enable_freshclam` than an additional parameter and a simp_options update.  It doesn't seem like a Stroolean would be appropriate in this case—`Enum['none', 'rsync', 'freshclam']` cover each case adequately, plus `true` is ambiguous and `false` seems redundant with `'none'`.

However: this update is intended to allow _existing SIMP 6.0.X customers_ access to a fix without major meddling with an API that should probably be more extensively refactored.

Reply:
@op-ct Fair enough. 👍